src/main/java/com/es/phoneshop/model/SortField.java

```diff
@@ -0,0 +1,6 @@
+package com.es.phoneshop.model;
```

Comment:
Move enum classes from model package to enum package and place it in the root alongside with other packages